controllers/notification/notification_controller.go

```diff
@@ -17,6 +16,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/builder"
 	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+	logger "sigs.k8s.io/controller-runtime/pkg/log"
```

Comment:
you didn't use this `logger` alias in your other PRs, so is it really needed here? (simply for the sake of consistency)

Reply:
that's definitely an error. great catch! let me fix that